parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -266,6 +266,68 @@ configureCNIIPTables() {
     fi
 }
 
+disableNtpAndTimesyncdInstallChrony() {
```

Comment:
Move to cse_install_ubuntu.sh. We will ask Mariner team to help on the Mariner version.

Reply:
FYI @hbeberman